src/test/java/org/folio/rest/utils/TestEntities.java

```diff
@@ -1,39 +1,17 @@
 package org.folio.rest.utils;
 
-import org.folio.rest.jaxrs.model.Adjustment;
 import org.folio.rest.jaxrs.model.Alert;
-import org.folio.rest.jaxrs.model.Claim;
-import org.folio.rest.jaxrs.model.Cost;
-import org.folio.rest.jaxrs.model.Details;
-import org.folio.rest.jaxrs.model.Eresource;
-import org.folio.rest.jaxrs.model.FundDistribution;
-import org.folio.rest.jaxrs.model.License;
-import org.folio.rest.jaxrs.model.Location;
-import org.folio.rest.jaxrs.model.Physical;
 import org.folio.rest.jaxrs.model.Piece;
 import org.folio.rest.jaxrs.model.PoLine;
 import org.folio.rest.jaxrs.model.PurchaseOrder;
 import org.folio.rest.jaxrs.model.ReportingCode;
-import org.folio.rest.jaxrs.model.Source;
-import org.folio.rest.jaxrs.model.VendorDetail;
 
 public enum TestEntities {
-  ADJUSTMENT("/orders-storage/adjustments", Adjustment.class, "adjustment.sample", "credit", "1.5", 16),
```

Comment:
I believe the `.sample` files for deleted entities are not required any longer and can be deleted as well.